templates/blank/_Cargo.toml

```diff
@@ -9,6 +9,9 @@ edition = "2021"
 [dev-dependencies]
 #odra_test_dependency
 
+[build-dependencies]
+odra-build = { path = "../odra-build", version = "0.8.1" }
```

Comment:
This will not work. It should be injected based on the `-s` parameter in `cargo-odra`.

Reply:
ty